src/elements/icon/icon.visual.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { html } from 'lit';
+
+import { describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './icon.js';
+
+describe(`sbb-icon`, () => {
+  describeViewports({ viewports: ['small'] }, () => {
+    for (const size of ['small', 'medium', 'large']) {
+      it(
+        `icon=circle-information size=${size} ${visualDiffDefault.name}`,
+        visualDiffDefault.with(async (setup) => {
+          await setup.withFixture(html`<sbb-icon name="circle-information-${size}"></sbb-icon>`);
+        }),
+      );
+
+      it(
```

Comment:
You could take out the color test from the size loop as color is not dependant on size